test/queryClient.test.ts

```diff
@@ -32,4 +59,285 @@ describe('query client', () => {
     assert(result, 'cannot request query GET_INDEXER');
     deepAssert(result.data.indexer);
   }, 16000)
+
+  it('can query indexer delegator', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_INDEXER_DELEGATORS,
+      variables: { id:'0xCef192586b70e3Fc2FAD76Dd1D77983a30d38D04' },
+    });
+    assert(result, 'cannot request query GET_INDEXER_DELEGATORS');
+    deepAssert(result.data.indexer);
+    expect(result.data.indexer.delegations.nodes).toBeTruthy()
```

Comment:
This will never be reached because if delegations isn't truthy the request would fail.

Reply:
ok i can remove `.delegations.nodes`